config/routes.rb

```diff
@@ -19,6 +19,7 @@
 
   resources 'facilities'
   resources 'spaces'
+  get '/spaces/:id/edit/:field', to: 'spaces#edit_field', as: 'edit_field'
```

Comment:
We probably better group these "custom" routes, so it would be 
```
  resources 'spaces' do
    collection do
      post :upload_image # upload_image_spaces_path
      ...
    end
  end

```
Although maybe we should do that in a seperate PR?

Reply:
Agree.

Reply:
Will you be making a PR on that?